docs/en/compute/Spark.md

```diff
@@ -140,17 +141,39 @@ After Alluxio 1.8 (not included), users can encode the Zookeeper service address
 inside an Alluxio URI (see [details](#access-data-from-alluxio-in-ha-mode)).
 In this way, it requires no extra setup for Spark configuration.
 
-### Customize Alluxio User Properties for Spark Jobs
+### Customize Alluxio User Properties for Individual Spark Jobs
 
-To customize Alluxio client-side properties in a Spark job, see
-[how to configure Spark Jobs](Configuration-Settings.html#spark-jobs).
+Spark users can use pass JVM system properties to Spark jobs by adding `"-Dproperty=value"` to
+`spark.executor.extraJavaOptions` for Spark executors and `spark.driver.extraJavaOptions` for
```

Comment:
can we recommend `--driver-java-options` instead of `spark.driver.extraJavaOptions`? For spark shell and spark submit, `--driver-java-options` will always work.